slides/lesson12.md

```diff
@@ -102,6 +134,33 @@ console.log(names[1]); // David
 
 ---
 
+### getting the length of an array
+
+We can get the length of an array with the `.length` property:
+
+```js
+let names = ["Alice", "Bob", "Carol"];
+console.log(names.length); // 3
```

Comment:
We might need a small clarification here that the length is always +1 bigger than the index of the last position

Reply:
I'll make sure I'll mention it